### DISCUSSION TITLE
Why is `crypto_fee` required in Manual out_csv file?

### DISCUSSION THREAD

**Discussion by jmbiggs** [eprbell/dali-rp2#230] [Q&A] [answered]
I'm trying to get my manual csv files working, and have run into this roadblock with the out_csv file.

Unlike in the in_csv file where `crypto_fee` and `fiat_fee` are mutually exclusive, the out_csv file seems to require `crypto_fee` even when `fiat_fee` is present.  I tried putting __unknown into the `crypto_fee` field, but that didn't work either.

Am I missing something?  Do I need to manually calculate these fees?  If I put '0' in for the crypto fee, will it still take into account the fiat fee?

Thanks

**Comment by eprbell** [comment, marked as answer]
This topic [was discussed here](https://github.com/eprbell/rp2/issues/82), PTAL.

**Comment by jmbiggs** [comment]
Ah, I see!  Didn't even think to search in the other project.  Thank you.
